### DISCUSSION TITLE
Which cli options should we use when adding strict mode?

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible/ansible-lint#2530] [Polls]
We have to options for adding strict mode to ansible-lint:

* `-W` to match sphinx-build and python
* `-s, --strict` to matchs other tools like yamllint and also be easier to remember

**Comment by apatard** [comment]
Personaly, I don't care that much about which option but I've voted -s option for two reasons:
- From my small experience, ansible users often know nothing about python or sphinx-build,
- -W may remind people of tools or compilers like gcc which are using -W for warnings.

**Reply by ssbarnea** [reply]
See https://github.com/ansible/ansible-lint/pull/2533
